### ISSUE TITLE
FormRequest validated() triggers validation on each usage

### ISSUE THREAD

**Issue by pionl** [laravel/framework#42126]
- Laravel Version: 9.5.1
- PHP Version: 8.1

### Description:

👋

While using FormRequests method `validated` with a `$key` argument it triggers validation on each call. 

If the developer does not now this, it can cause some small performance issues. It would be great to call validate only once (cache the result) or not support `$key` property to prevent this issues.

Thank you and hope that description is OK.

### Steps To Reproduce:

1. Debug `Validator->validated()` method
2. Call `$request->validated('one'); request->validated('two')`

**Comment by driesvints** [comment]
Atm, this is the expected behavior. I suggest caching the result in userland if you need to.
